app/src/main/java/com/example/footballmanager/ui/theme/navigation/BottomNavigationBar.kt

```diff
@@ -97,4 +102,9 @@ fun ButtonNavigationBar(
 
         }
     }
+}
+
+@Preview
+@Composable
+fun fafa(){
```

Comment:
same

Reply:
giraffe fafafafa is tall like a wardrobe